ooiui/static/json/uFramePatchNotes.json

```diff
@@ -1,5 +1,17 @@
 [
     {
+        "VersionNumber":"1.2.4",
+        "VersionDate":"2017-04-12",
+        "VersionDescription":[
+            "Constrain time bounds for queries (11979)",
+	    "add parameter list to annotations (12130)",
```

Comment:
I know it is minor, but could you use capital letters to start and add a period at the end of each?